packages/dicomImageLoader/src/imageLoader/wadors/metaData/metaDataProvider.ts

```diff
@@ -99,14 +99,14 @@ function metaDataProvider(type, imageId) {
 
     if (imageOrientationPatient) {
       rowCosines = [
-        parseFloat(imageOrientationPatient[0]),
-        parseFloat(imageOrientationPatient[1]),
-        parseFloat(imageOrientationPatient[2]),
+        parseFloat(imageOrientationPatient[0] as any),
```

Comment:
getNumberValues already returns an array of number[], so you shouldn't need the parsefloat any longer, nor the cast as any.  If you do need the case, as any is a better cast than as string.  I suggest dropping the cast and letting typescript complain until the response type from getNumberValues is properly typed.